packages/api/amplify_api/example/lib/graphql_api_view.dart

```diff
@@ -44,13 +44,19 @@ class _GraphQLApiViewState extends State<GraphQLApiView> {
       onEstablished: () => print('Subscription established'),
     );
 
-    try {
-      await for (var event in operation) {
-        print('Subscription event data received: ${event.data}');
-      }
-    } on Exception catch (e) {
-      print('Error in subscription stream: $e');
-    }
+    final streamSubscription = operation.listen(
+      (event) {
+        final result = 'Subscription event data received: ${event.data}';
+        print(result);
```

Comment:
Can you make sure the example app is using `package:amplify_lints/app.yaml`?

Reply:
It's not, and there would be quite a few changes needed to make it comply so will do as a separate PR.